plugins/modules/win_domain.ps1

```diff
@@ -9,13 +9,26 @@ $ErrorActionPreference = "Stop"
 
 # FUTURE: Consider action wrapper to manage reboots and credential changes
 
+$required_features = @("AD-Domain-Services","RSAT-ADDS", "RSAT-AD-AdminCenter")
+
+Function Get-MissingFeatures {
+    $features = @(Get-WindowsFeature $required_features)
```

Comment:
Instead of referencing `$required_features` as a var outside of this function you should pass in a list of features where it is called.

Reply:
Changed in both `win_domain` and `win_domain_controller`.